src/Components/Title.css

```diff
@@ -0,0 +1,23 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.title-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  text-align: center;
+}
+
+.title-container h1 {
+  font-size: 4rem;
+  padding-bottom: 10px;
+}
+
+.title-container p {
+  font-size: 1.2rem;
+  padding-bottom: 50px;
+}
```

Comment:
The title component is being developed on the Login UI ticket which is Annie's ticket. Please remove this file and revert the Title component back to how it was. If you want, you can share these as snippets to Annie.

Reply:
Please remove this from this PR.